core/includes/database/database.inc

```diff
@@ -826,13 +853,13 @@ abstract class DatabaseConnection {
   /**
    * Prepares and returns a SELECT query object.
    *
-   * @param $table
+   * @param string $table
    *   The base table for this query, that is, the first table in the FROM
    *   clause. This table will also be used as the "base" table for query_alter
    *   hook implementations.
-   * @param $alias
+   * @param string $alias
    *   The alias of the base table of this query.
-   * @param $options
+   * @param array $options
    *   An array of options on the query.
    *
    * @return SelectQuery|stdClass[]
```

Comment:
I wonder under which circumstances we'd end up with an array of objects here.

Reply:
I've removed it, it doesn't seem applicable.

Reply:
> I've removed it, it doesn't seem applicable.

Probably not committed yet, as the `stdClass[]` type's still there. Or is this "removed" related to something else?